pom.xml

```diff
@@ -224,12 +224,21 @@
           <configuration>
```

Comment:
Did you plan to leave the Maven build-system in place?

Reply:
Only for the moment; once the CI and releaser tasks are converted, the Maven build scripts can be removed.